sdk/python/kfp/cli/run.py

```diff
@@ -185,13 +209,20 @@ def _display_run(client, namespace, run_id, watch, output_format, detail=False):
         _print_runs([run], output_format)
 
 
-def _wait_for_run_completion(client, run_id, timeout, output_format):
+def _wait_for_run_completion(client: Client, run_id: str, timeout: int,
+                             output_format: OutputFormat):
     run_detail = client.wait_for_run_completion(run_id, timeout)
     _print_runs([run_detail.run], output_format)
 
 
-def _print_runs(runs, output_format):
-    headers = ['run id', 'name', 'status', 'created at']
-    data = [[run.id, run.name, run.status,
-             run.created_at.isoformat()] for run in runs]
+def _print_runs(runs: List[kfp_server_api.ApiRun], output_format: OutputFormat):
+    headers = ['run id', 'name', 'status', 'created at', 'experiment id']
+    data = [[
+        run.id, run.name, run.status,
+        run.created_at.isoformat(),
+        next(rr
+             for rr in run.resource_references
+             if rr.key.type == kfp_server_api.ApiResourceType.EXPERIMENT).id
+    ]
+            for run in runs]
```

Comment:
I tried to uncomment these lines and the error no longer appears.
Can you double check if sth went wrong here?

Reply:
Ohh, I think the fix is: ").key.id"

Reply:
Sent #6997 to fix this